satpy/tests/reader_tests/test_fci_l2_nc.py

```diff
@@ -507,3 +507,90 @@ def test_byte_extraction(self):
                                                 })
 
         assert dataset.values == 0
+
+class TestFciL2NCAMVFileHandler(unittest.TestCase):
+    """Test the FciL2NCFileHandler reader."""
+
+    def setUp(self):
+        """Set up the test by creating a test file and opening it with the reader."""
+        # Easiest way to test the reader is to create a test netCDF file on the fly
+        # Create unique filenames to prevent race conditions when tests are run in parallel
+        self.test_file = str(uuid.uuid4()) + ".nc"
```

Comment:
I would suggest using pytest's `tmp_path` fixture instead. That will also allow removing the teardown method.

Reply:
Do you have an example of that ?

Reply:
Here is an example of using `tmp_path_factory` within a fixture for a similar test: https://github.com/pytroll/satpy/blob/main/satpy/tests/reader_tests/test_sgli_l1b.py#L21-L47
More info on tmp_path: https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

Reply:
that is fine with me.